product_docs/docs/biganimal/release/overview/02_high_availability.mdx

```diff
@@ -80,4 +80,4 @@ A single-region configuration with three data nodes (one lead and two shadow nod
 
 For instructions on creating a distributed high-availability cluster using the BigAnimal portal, see [Creating a distributed high-availability cluster](../getting_started/creating_a_cluster/creating_an_eha_cluster/).
 
-For instructions on creating, retrieving information from, and managing a distributed high-availability cluster using the BigAnimal CLI, see [Using `pgd` commands](/biganimal/latest/reference/cli/managing_clusters/#using-pgd-commands).
+For instructions on creating, retrieving information from, and managing a distributed high-availability cluster using the BigAnimal CLI, see [Using `pgd` commands](/biganimal/latest/reference/cli/managing_clusters/#managing-distributed-high-availability-clusters).
```

Comment:
Should the section name be changed here as well, to match the link? It still says **Using pgd commands**.